android/lib/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.airbnb.android.react.maps" >
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
Nice add for the permission check. Was it intended to in the same PR as "Fixes lint errors" ?